dali/test/cv_mat_utils.h

```diff
@@ -46,6 +48,41 @@ cv::Mat_<cv::Vec<T, nchannels>> copy_to_mat(Box<2, int> roi, T *base, int rows,
   return out_copy;
 }
 
+
+/**
```

Comment:
What is a rationale behind that change. You missed it in the PR description.

Reply:
Sorry, missed that. Updated the description:
>Also, there was one function, that might come in handy in future (copying cv::Mat onto GPU), I put it in cv_mat_utils file, where it belongs